logging/docs/conf.py

```diff
@@ -20,12 +20,12 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath(".."))
 
-__version__ = "0.91.4"
+__version__ = "0.1.0"
```

Comment:
Shouldn't block this PR, but I'm pretty sure we should be using the current version from each library's `setup.py` here.